### DISCUSSION TITLE
debug-mode="requests" and print2 function

### DISCUSSION THREAD

**Discussion by nobanzai** [gerbera/gerbera#3268] [General]
Is it intended, that debug-mode="requests" leads to

`info: [js_functions.cc:61] js_print2():
`

for **all** print2 calls?

**Comment by KarlStraussberger** [comment]
`print2` needs two arguments. Level and your message. Check your code.

**Comment by nobanzai** [comment]
I always use print2("Info", "some text" );
And the message mentioned above is shown only when using debug-mode="requests".
So IMHO this can't be a syntax problem.

**Reply by KarlStraussberger** [reply]
If you do not set debug-mode the format of the message is different. As soon as some debugging mode is active we print line numbers etc with each message.

**Reply by nobanzai** [reply]
This is a bit confusing, because output is different in a context, that wasn't meant to be debugged.
But ok, I can live with that.
